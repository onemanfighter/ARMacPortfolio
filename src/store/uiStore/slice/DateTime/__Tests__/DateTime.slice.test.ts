import { renderHook } from '@testing-library/react';
import { uiStore } from '@uiStore';

const RealDate = Date.now;

describe('Date time slice', () => {
  beforeAll(() => {
    // Lock Time
    global.Date.UTC = jest.fn().mockImplementation();
  });

  it.skip('should return date time selector state and actions', () => {
    jest.setSystemTime(new Date('2020-01-01'));
    const dateTime = renderHook(() => uiStore()).result.current;

    expect(dateTime.DateTime).toMatchSnapshot();
  });
});
