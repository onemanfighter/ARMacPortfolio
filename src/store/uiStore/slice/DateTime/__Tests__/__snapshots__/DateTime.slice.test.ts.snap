// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Date time slice should return date time selector state and actions 1`] = `
{
  "date": 2024-05-29T21:05:29.050Z,
  "init": [Function],
}
`;
